.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
```

Comment:
This file probably needs to be removed (or replaced with Woodpecker / Drone in a different PR)?

Reply:
Oi I didn't realise Ember generated this file during the upgrade. I don't think we use a set-up like that on this repo, the hosting is done through Github itself, via the gh-pages branch.

Reply:
It could still be useful for running the linters.